### DISCUSSION TITLE
TandoorRecipes on a QNAP NAS

### DISCUSSION THREAD

**Discussion by mastles** [TandoorRecipes/recipes#1376] [Q&A]
My situation: 
I just got an QNAP TS-230 NAS.
I want to run TandoorRecipes in its so-called Container Station. That's designed for running Docker containers.

Has anyone of you experience with running TandoorRecipes on a QNAP NAS?

**Comment by Bartliwaggis** [comment]
Same with me (QNAP TS-251D here) - I also would like to run it on QNAP Container Station. 
A propriate docker image (x86) would be nice. Thanks for any help.

**Reply by smilerz** [reply]
X86 container already exists.

**Reply by Bartliwaggis** [reply]
Great - I just discovered your (beta) image and install it right now. Thanks a lot!

**Reply by smilerz** [reply]
Beta is unstable. I recommend latest.
